src/server/gc_worker/gc_worker.rs

```diff
@@ -2614,16 +2637,13 @@ mod tests {
         assert!(ks.is_empty());
     }
 
-    // setup engine and prepare some data:
-    //  three regions:
-    //  region 1: includes ("k00", "value-00") to ("k09", "value-09")
-    //  region 2: includes ("k10", "value-10") to ("k19", "value-19")
-    //  region 3: includes ("k20", "value-20") to ("k29", "value-29")
+    #[allow(non_snake_case)]
```

Comment:
Which part violates this rule?

Reply:
<img width="965" alt="image" src="https://user-images.githubusercontent.com/97348524/194818627-d3bc5b27-47dd-4be1-8b7e-be91bd87f651.png">

Reply:
Which toolchain are you using? This should be fixed in https://github.com/rust-lang/rust/pull/89473 last year.

Reply:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/97348524/194830529-3b3bfd69-68d1-422a-a9d2-e8afd26f59c4.png">

Reply:
Oh, I see it now. You need to add an extra `_ => (*cf, RocksCfOptions::default()),`, right now rust mistakenly treats `CF_RAFT` as a new identifier instead of the constant.

Reply:
<img width="892" alt="image" src="https://user-images.githubusercontent.com/97348524/194843421-328f474b-9dee-4d98-af03-4f87aa0cb027.png">
i remember I tried your suggestion already, and above the other warning. So, It should be the rustc issue anyway.

Reply:
Need to `use engine_traits::CF_RAFT`.